CODEOWNERS

```diff
@@ -22,12 +22,14 @@ scripts/codegen-translations.js @jinchung @marco
 src/apollo @brunobar79 @jinchung @derHowie
 
 # ASSETS
-src/assets @skylarbarrera 
+src/assets @skylarbarrera
 
 # COMPONENTS
 src/components @osdnk @terrysahaidak
 src/components/unique-token @jkadamczyk
 src/components/floating-emojis @jkadamczyk
+src/components/buttons/hold-to-authorize @jkadamczyk
```

Comment:
I have some opinions about this https://rainbowhaus.slack.com/archives/CGUDDJYNL/p1652689873841879

Reply:
You're right about it I will leave both you and Terry there next to me.